lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -0,0 +1,231 @@
+import datetime
+
+import pytest
+import responses
+from click.testing import CliRunner
+
+from pbench.cli.server.user_management import (
+    USER_LIST_HEADER_ROW,
+    USER_LIST_ROW_FORMAT,
+    user_create,
+    user_delete,
+    user_list,
+    user_update,
+)
+
+
+class TestUserManagement:
+    USER_SWITCH = "--username"
+    PSWD_SWITCH = "--password"
+    PSWD_PROMPT = "Password: "
+    EMAIL_SWITCH = "--email"
+    FIRST_NAME_SWITCH = "--first-name"
+    LAST_NAME_SWITCH = "--last-name"
+    ROLE_SWITCH = "--role"
+    USER_TEXT = "test_user"
+    PSWD_TEXT = "password"
+    EMAIL_TEXT = "test@domain.com"
+    FIRST_NAME_TEXT = "First"
+    LAST_NAME_TEXT = "Last"
+
+    @staticmethod
+    @responses.activate
+    def test_help(pytestconfig):
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(user_create, ["--help"])
+        assert result.exit_code == 0
+        assert str(result.stdout).startswith("Usage:")
+        assert not result.stderr_bytes
+
+    @staticmethod
+    def register_user():
```

Comment:
This used to be a test, and you refactored it into a helper function...and, in the process you deleted the test.  Are you assuming that it's use by the other tests is sufficient?

Reply:
This helper function also internally tests for any errors

Reply:
Not sure we want helper functions performing tests.

Reply:
> Not sure we want helper functions performing tests.
I could go either way on that.  I prefer that the code be DRY, so I have no problem with avoiding duplicating assertions by placing them in the helper function.

On the other hand, I can see a line of reasoning which states that helper functions should be supporting tests and not performing them.  So, arguably, there should be a test which calls the helper and then performs the assertions to make sure that the functions performed by the helper operated properly; then, in subsequent tests, those assertions _should_ be redundant, so there is no need to perform them in the helper (and presumably no need to perform them in the subsequent tests, either), and the subsequent tests and just presume that the helper did its work.